tensorboard/plugins/example/tensorboard_plugin_example/demo.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Demo code."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from absl import app
+import tensorflow as tf
+
+from tensorboard_plugin_example import summary_v2
+
+
+tf.compat.v1.enable_eager_execution()
+tf = tf.compat.v2
+
+
+def main(unused_argv):
+  writer = tf.summary.create_file_writer("demo_logs")
+  with writer.as_default():
+    summary_v2.greeting(
+        "guestbook",
+        "Alice",
+        step=0,
+        description="Sign your name!",
+    )
+    summary_v2.greeting("guestbook", "Bob", step=1)  # no need for `description`
+    raw_pb = summary_v2.greeting_pb("guestbook", "Cheryl")
+    tf.summary.experimental.write_raw_pb(
```

Comment:
Hmm, okay—do we still encourage plugins to define `pb` functions? If so,
would you prefer that we leave this `pb` function untested, or is there
an alternate path (using the TF 1.x writer after reinterpret-casting the
protos doesn’t seem much nicer)?

Reply:
I'm kind of inclined to just omit the `pb` function for the example for now.  I think in TF 2 there is honestly not much reason to write proto summaries via TF APIs (as opposed to via APIs that are entirely TF-free, like tensorboardX and the proposed new TB summary API).  The point in TF 1 was to bypass the overhead of building a graph and constructing a session (since the file writing worked without any of that).  But in TF 2, the file writing is op-based anyway; the `write_raw_pb()` API doesn't avoid that.

The purpose I had in adding `write_raw_pb()` was pretty much purely to aid in TF 2 migration for existing TF code that emitted serialized custom summary protos (since otherwise, there isn't an easy way to do an in-graph deconstruction of the protos).  New summary ops should be defined using the `write()` API.

Reply:
Got it; thanks for the context. Removed, and trimmed down `metadata.py`
accordingly.